library/src/main/java/pyxis/uzuki/live/naraeimagepicker/fragment/ImageFragment.kt

```diff
@@ -47,7 +52,25 @@ class ImageFragment : BaseFragment<ImageItem>() {
             if (recyclerView != null) {
                 recyclerView.notifyDataSetChanged()
             }
-            sendEvent(ToolbarEvent("$mAlbumName (${itemList.size})", true))
+            sendEvent(ToolbarEvent("$mAlbumName ($selectCount)", true))
+        }
+    }
+
+    private fun onImageClick(imageItem: ImageItem) {
+        if (SelectedItem.contains(imageItem)) {
+            SelectedItem.removeItem(imageItem)
+            selectCount--
+        } else {
+            selectCount++
+            addSelectedItem(imageItem)
+        }
+        sendEvent(ToolbarEvent("$mAlbumName($selectCount)", true))
```

Comment:
You don't need to store 'selectCount' as global variable. 
So, i recommend reflect these changes.

1. implement `size` in SelectedItem (methods or variables)
2. send ToolbarEvent by ```$mAlbumName(${SelectedItem.size})``` instead of ```$mAlbumName($selectCount)```

this changes effect both fragment, AllFragment and ImageFragment.

I think it will take longer than the scheduled time, So if the changes are reflected, i will publish temporary version.
In future version, implementation methods may changed.